CHANGELOG.md

```diff
@@ -0,0 +1,28 @@
+## 1.5
+
+### 1.5 - 2021-12-01
+
+#### Enhancements made
+
+- Support recent rstudio-server releases.
+
+#### Dependency updates
+
+* Bump jupyter-server-proxy from 3.1.0 to 3.2.0.
+* Remove check for RSESSION_PROXY_RSTUDIO_1_4 environment variable. jupyter-rsession-proxy now supports later versions of rstudio-server 1.4 and the newer date-based releases. For older versions of rstudio-server, please use jupyter-rsession-proxy <= 1.4.
```

Comment:
Let's pull this out as a BREAKING CHANGE?

Reply:
Do you mean isolate it in the changelog in a new sectioni, or actually remove it from the release?

Reply:
@ryanlovett I think we should add a new heading to the changelog, as well as make this 2.0 - to make it super clear that you can't just upgrade this package without also upgrading rstudio.

Reply:
Updated. How does it look now?